syntax = "proto3";

message AlertRequest {
  int64 serviceId = 1;
}

message ReceiptConfirmation {
  int64 serviceId = 1;
}

message Status {
  bool okay = 1;
  string message = 2;
}

service AlertManager {
  rpc Alert(AlertRequest) returns (Status) {}
  rpc handleReceiptConfirmation(ReceiptConfirmation) returns (Status) {}
}